lib/helpers/cli.js

```diff
@@ -221,6 +221,10 @@ export function getPossibleOptionNames(specifiedOptions) {
 function executeGlobby(workingDir, pattern, ignore) {
   let supportedExtensions = new Set(['.hbs', '.handlebars']);
 
+  if (path.extname(pattern) === '.svg') {
```

Comment:
Hmm, I don't think that treating a regexp pattern as a path is robust enough. A regexp pattern doesn't necessarily look like a path, so while this might work sometime, it isn't really a reliable way to do this.